user_tools/src/spark_rapids_pytools/resources/qualification-conf.yaml

```diff
@@ -81,6 +87,8 @@ local:
       - 'Estimated App Cost'
       - 'Estimated GPU Cost'
       - 'Estimated GPU Savings(%)'
+      - 'Job Frequency(monthly)'
+      - 'Annual Cost Savings($)'
```

Comment:
nit: we probably don't need `($)`. Other absolute cost columns like `Estimated App /GPUCost` don't show the unit.

Reply:
Fixed in latest commit